pkg/controller/certificatemanager/certificatemanager.go

```diff
@@ -85,14 +85,18 @@ type CertificateManager interface {
 
 // Create creates a signer of new certificates and has methods to retrieve existing KeyPairs and Certificates. If a user
 // brings their own secrets, CertificateManager will preserve and return them.
-func Create(cli client.Client, installation *operatorv1.InstallationSpec, clusterDomain string) (CertificateManager, error) {
+func Create(cli client.Client, installation *operatorv1.InstallationSpec, clusterDomain, ns string) (CertificateManager, error) {
```

Comment:
This function now takes a namespace.

For single-tenant clusters, this namespace will be passes a "tigera-operator"

For multi-tenant clusters, this will be the tenant's namespace.